homeassistant/components/lg_thinq/sensor.py

```diff
@@ -255,9 +255,77 @@
         translation_key=ThinQProperty.WATER_TYPE,
     ),
 }
+ELAPSED_DAY_SENSOR_DESC: dict[ThinQProperty, SensorEntityDescription] = {
+    ThinQProperty.ELAPSED_DAY_STATE: SensorEntityDescription(
+        key=ThinQProperty.ELAPSED_DAY_STATE,
+        native_unit_of_measurement=UnitOfTime.DAYS,
+        translation_key=ThinQProperty.ELAPSED_DAY_STATE,
+    ),
+    ThinQProperty.ELAPSED_DAY_TOTAL: SensorEntityDescription(
+        key=ThinQProperty.ELAPSED_DAY_TOTAL,
+        native_unit_of_measurement=UnitOfTime.DAYS,
+        translation_key=ThinQProperty.ELAPSED_DAY_TOTAL,
+    ),
+}
+TIME_SENSOR_DESC: dict[ThinQProperty, SensorEntityDescription] = {
+    TimerProperty.LIGHT_START: SensorEntityDescription(
+        key=TimerProperty.LIGHT_START,
+        translation_key=TimerProperty.LIGHT_START,
+    ),
+    TimerProperty.ABSOLUTE_TO_START: SensorEntityDescription(
+        key=TimerProperty.ABSOLUTE_TO_START,
+        translation_key=TimerProperty.ABSOLUTE_TO_START,
+    ),
+    TimerProperty.ABSOLUTE_TO_STOP: SensorEntityDescription(
+        key=TimerProperty.ABSOLUTE_TO_STOP,
+        translation_key=TimerProperty.ABSOLUTE_TO_STOP,
+    ),
+}
+TIMER_SENSOR_DESC: dict[ThinQProperty, SensorEntityDescription] = {
+    TimerProperty.RELATIVE_TO_START: SensorEntityDescription(
+        key=TimerProperty.RELATIVE_TO_START,
+        translation_key=TimerProperty.RELATIVE_TO_START,
+    ),
+    TimerProperty.RELATIVE_TO_START_WM: SensorEntityDescription(
+        key=TimerProperty.RELATIVE_TO_START,
+        translation_key=TimerProperty.RELATIVE_TO_START_WM,
+    ),
+    TimerProperty.RELATIVE_TO_STOP: SensorEntityDescription(
+        key=TimerProperty.RELATIVE_TO_STOP,
+        translation_key=TimerProperty.RELATIVE_TO_STOP,
+    ),
+    TimerProperty.RELATIVE_TO_STOP_WM: SensorEntityDescription(
+        key=TimerProperty.RELATIVE_TO_STOP,
+        translation_key=TimerProperty.RELATIVE_TO_STOP_WM,
+    ),
+    TimerProperty.SLEEP_TIMER_RELATIVE_TO_STOP: SensorEntityDescription(
+        key=TimerProperty.SLEEP_TIMER_RELATIVE_TO_STOP,
+        translation_key=TimerProperty.SLEEP_TIMER_RELATIVE_TO_STOP,
+    ),
+    TimerProperty.REMAIN: SensorEntityDescription(
+        key=TimerProperty.REMAIN,
+        translation_key=TimerProperty.REMAIN,
+    ),
+    TimerProperty.TARGET: SensorEntityDescription(
+        key=TimerProperty.TARGET,
+        translation_key=TimerProperty.TARGET,
+    ),
+    TimerProperty.RUNNING: SensorEntityDescription(
+        key=TimerProperty.RUNNING,
+        translation_key=TimerProperty.RUNNING,
+    ),
+    TimerProperty.TOTAL: SensorEntityDescription(
+        key=TimerProperty.TOTAL,
+        translation_key=TimerProperty.TOTAL,
+    ),
+}
```

Comment:
These should have the device class timestamp and not write state every time. We should calculate when it happens and make that the state

Reply:
Added device class DURATION for continuously changed timer and collect that by TIMER_SENSOR_DESC.

Reply:
> continuously changed timer

How do you mean that? What are those?

Reply:
We update the commit that add the device class for do not write state per minutes, as you commented.